src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const population = document.getElementsByClassName('population');
+const arr = [];
+const texts = [...population].map(item => item.innerText);
+
+for (let i = 0; i < texts.length; i++) {
+  arr.push(+texts[i].split(',').join(''));
+}
```

Comment:
```suggestion
const arr = [...population].map(item => +item.innerText.split(',').join(''));
```
you can make all it in map iteration.
something like that